parcel_test.go

```diff
@@ -31,57 +31,106 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	parcel.Number, err = store.Add(parcel)
+
+	require.NoError(t, err)
+	require.NotEmpty(t, parcel.Number)
 
 	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	getParcel, err := store.Get(parcel.Number)
+
+	require.NoError(t, err)
+	require.Equal(t, parcel, getParcel)
 
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что посылку больше нельзя получить из БД
+	deleteParcel := store.Delete(parcel.Number)
+
+	require.NoError(t, deleteParcel)
+	_, err = store.Get(parcel.Number)
+
+	require.Equal(t, sql.ErrNoRows, err)
 }
 
 // TestSetAddress проверяет обновление адреса
 func TestSetAddress(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+
+	defer db.Close()
+
+	store := NewParcelStore(db)
+	parcel := getTestParcel()
 
 	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	parcel.Number, err = store.Add(parcel)
+
+	require.NoError(t, err)
+	require.NotEmpty(t, parcel.Number)
 
 	// set address
-	// обновите адрес, убедитесь в отсутствии ошибки
 	newAddress := "new test address"
 
+	setAddress := store.SetAddress(parcel.Number, newAddress)
```

Comment:
Лучше ошибки именовать везде одинаково, а то не очень очевидно, что `setAddress` это ошибка
```suggestion
err = store.SetAddress(parcel.Number, newAddress)
```

Reply:
исправлено